namer/dnssrv/src/main/scala/io/buoyant/namer/dnssrv/DnsSrvNamer.scala

```diff
@@ -32,80 +30,84 @@ class DnsSrvNamer(
   private val memoizedLookup: (Path) => Activity[NameTree[Name]] = Memoize { path =>
     path.take(1) match {
       case id@Path.Utf8(address) =>
-        Activity(Var.async[State[NameTree[Name]]](Activity.Pending) { state =>
+        val vaddr = watchDns(address).run.map {
+          case Activity.Ok(rsp) =>
+            Addr.Bound(rsp: _*)
+          case Activity.Pending =>
+            Addr.Pending
+          case Activity.Failed(e) =>
+            log.debug("SDS lookup failure: %s", e.getMessage)
```

Comment:
maybe replace `SDS` with `DNS-SRV` for consistency? I've only run across the term SDS in Envoy docs, but I don't think it applies here.